Restructured page "system events" and replaced lists with tables for better overview (#179)
emnify/product-docs: docs/services/platform/events/index.md

You can find detailed information on the Event Object and its associated properties in the [Event API reference](https://cdn.emnify.net/api/doc/event.html) in the _emnify System Documentation_.  
:::

### Event type

> Here you can find an [overview of the specific events](/system-events/event-types) generated by our platform.

Reviewer: ```suggestion
For an overview of the specific events generated by our platform, see [Event types](/system-events/event-types).
```

nit: altered the sentence structure to more closely follow the [Google developer documentation style guide](https://developers.google.com/style/cross-references?hl=en)

Author: Oh that's a lot better! Thank you
